docs/reference/commandline/service_create.md

```diff
@@ -1120,6 +1121,59 @@ $ docker service create --name cuda \
                         nvidia/cuda
 ```
 
+### Running as a job
+
+Jobs are a special kind of service designed to run an operation to completion
+and then stop, as opposed to running long-running daemons. When a Task
+belonging to a job exits successfully (return value 0), the Task is marked as
+"Completed", and is not run again.
+
+Jobs are started by using one of two modes, `replicated-job` or `global-job`
+
+```bash
+$ docker service create --name myjob \
+                        --mode replicated-job \
+                        bash "true"
+```
+
+This command will run one Task, which will, using the `bash` image, execute the
+command `true`, which will return 0 and then exit.
+
+Importantly, none of the update or rollback configuration options are valid.
+Jobs can be updated, but cannot be rolled out or rolled back.
+
+Jobs are available in both replicated and global modes.
+
+#### Replicated Jobs
+
+A replicated job is like a replicated service. Setting the `--replicas` flag
+will specify total number of iterations of a job to execute.
+
+By default, all replicas of a replicated job will launch at once. To control
+the total number of replicas that are executing simultaneously at any one time,
+the `--max-concurrent` flag can be used:
+
+```bash
+$ docker service create --name mythrottledjob \
+                        --mode replicated-job \
```

Comment:
Maybe `--kind=job --mode=replicated`?

Reply:
Alternatively this could be `docker job create` even if the API is for a service.

Reply:
While `docker job create` is probably a clean(er); advantages;

- separate subcommand
- we could hide/remove flags that don't apply to jobs

Downside:

- given that they're both backed by a service, we need to either filter out jobs from `docker service ls` (etc) and vice-versa.
- that might become confusing if we don't apply the same filter very strictly (`docker service rm <job service>` could otherwise remove a job)
- also think of `docker service create myjob`, which could show an error that `service myjob already exists`, but `docker service ls` wouldn't show it